Blockzilla/AppDelegate.swift

```diff
@@ -280,8 +280,11 @@ extension AppDelegate {
         
         Telemetry.default.beforeSerializePing(pingType: CorePingBuilder.PingType) { (inputDict) -> [String : Any?] in
             var outputDict = inputDict // make a mutable copy
-            
-            outputDict["showTrackerStatsShare"] =  self.browserViewController.shouldShowTrackerStatsShareButton()
+
+            if self.browserViewController.canShowTrackerStatsShareButton() {
+                _ = self.browserViewController.shouldShowTrackerStatsShareButton() // Force a coin flip
+                outputDict["showTrackerStatsShare"] = UserDefaults.standard.bool(forKey: BrowserViewController.userDefaultsShareTrackerStatsKeyNEW)
```

Comment:
Sorry about the confusion here - the outputDict key should be changed, so that our data folks have clear separation from Phase I of the experiment and this Phase II.

Reply:
I also think the bool should be set to 'canShowTrackerStatsShareButton()', as we want that to be determined once and then re-used.

Reply:
@Sdaswani am I following this right: 
 - If the app is klar or user is non en-us then nothing is added to `outputDict`
 - Otherwise we are adding a value to `outputDict` to indicate the value of the coin flip. It does not indicate if they have enough trackers blocked to actually show it.

Reply:
* Yes, those type of users shouldn't even coin flip.
* Yes, exactly, as even though we aren't _showing_ the trackers blocked, we want to know which users may get the messaging in the (_hopefully_) near future.

Reply:
Ok I think this logic was mostly correct but pretty unclear. Hopefully the changes I've made clear it up a bit.